morph_tool/transform.py

```diff
@@ -98,3 +105,119 @@ def align(section, direction):
     matrix = Rotation.from_rotvec(alpha * axis).as_matrix()
 
     rotate(section, matrix, origin=section.points[0])
+
+
+def rotation_matrix_from_vectors(vec1, vec2):
+    """ Find the rotation matrix that aligns vec1 to vec2
+
+    Picked from: https://stackoverflow.com/a/59204638/3868743
+
+    :param vec1: A 3d "source" vector
+    :param vec2: A 3d "destination" vector
+    :return mat: A transform matrix (3x3) which when applied to vec1, aligns it with vec2.
+    """
+    a, b = (vec1 / np.linalg.norm(vec1)).reshape(3), (vec2 / np.linalg.norm(vec2)).reshape(3)
+    v = np.cross(a, b)
+    c = np.dot(a, b)
+    s = np.linalg.norm(v)
+    if s == 0:
+        return np.eye(3)
+    kmat = np.array([[0, -v[2], v[1]], [v[2], 0, -v[0]], [-v[1], v[0], 0]])
+    rotation_matrix = np.eye(3) + kmat + kmat.dot(kmat) * ((1 - c) / (s ** 2))
+    return rotation_matrix
+
+
+# pylint: disable=too-many-branches
+def align_pyr_morph(morph, direction=None, method='whole_apical', apical_point=None):
```

Comment:
let's add an option that if method = ndarray, we use this as direction of pia to apply the rotation, so we can bypass any search algo we have